guides/source/debugging_rails_applications.md

```diff
@@ -232,23 +241,41 @@ irb(main):003:0> Article.pamplemousse
 => #<Comment id: 2, author: "1", body: "Well, actually...", article_id: 1, created_at: "2018-10-19 00:56:10", updated_at: "2018-10-19 00:56:10">
 ```
 
-Below each database statement you can see arrows pointing to the specific source filename (and line number) of the method that resulted in a database call. This can help you identify and address performance problems caused by N+1 queries: single database queries that generates multiple additional queries.
+Below each database statement you can see arrows pointing to the specific source filename (and line number) of the method that resulted in a database call e.g. `↳ app/models/article.rb:5`. This can help you identify and address performance problems caused by N+1 queries: single database queries that generates multiple additional queries.
 
 Verbose query logs are enabled by default in the development environment logs after Rails 5.2.
 
+```rb
+# config/application.rb
+# or any environment file:
+config.active_record.verbose_query_logs = true
+```
+
+```irb
+# bin/rails console
+ActiveRecord.verbose_query_logs = true
+```
+
```

Comment:
I think what we want to do here is just link to the config page:
`configuring.html#config-active-record-verbose-query-logs`

And we should remove the sentence above this about "after Rails 5.2" it is not relevant anymore.

The important thing is that it is only enabled by default in development.

Reply:
Applied.